src/_companies/europe/germany/saxony/pludoni/index.md

```diff
@@ -0,0 +1,19 @@
+---
+name:  "pludoni GmbH"
+continent: "Europe"
+country: "Germany"
+region: "Saxony"
+city: "Dresden"
+website: https://www.pludoni.de/
+careers_page: https://www.pludoni.de/jobs
+stack:
+  - "rails"
+  - "vuejs"
+created_at: 2023-03-19 23:45:16
+logo: logo.png
+work_arrangements: [remote,hybrid]
+---
+
+Pludoni GmbH is a small, self-funded SaaS provider specializing in recruitment. Founded in 2009, we operate several regional job boards under our Germany-wide brand Empfehlungsbund, and offer additional recruitment services such as Applicant Tracking Systems, recruitment control, and employer branding. Our unique selling proposition (USP) is our recommendation system. All participating recruiters can recommend qualified candidates who were not hired due to factors unrelated to their skills, such as reluctance to relocate, aversion to remote work, or mismatched salary expectations. These candidates receive a recommendation code that they can use on our platform to apply to other companies, along with a letter of recommendation from the recruiter. Leveraging Rails and other open-source technologies, we deliver a competitive platform with a small team of engineers.
```

Comment:
Could you change the description text to be in the third person? I'd like to keep this consistent for all descriptions.